### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by hugok79** [SubtitleEdit/subtitleedit#6228] [General]
@niksedk From which we should translate now?

[LanguageBaseEnglish.xml](https://github.com/SubtitleEdit/subtitleedit/blob/master/LanguageBaseEnglish.xml)
[LanguageMaster.xml](https://github.com/SubtitleEdit/subtitleedit/blob/master/LanguageMaster.xml)

**Comment by niksedk** [comment]
It should be the `LanguageBaseEnglish.xml` now :)
